theme/templates/pages/menus/dropdown.html

```diff
@@ -15,11 +93,9 @@
         <a href="https://www.hydroshare.org/apps/">Apps</a>
     </li>
     <li class="" id="dropdown-menu-https:--help.hydroshare.org">
-        <a href="http://help.hydroshare.org">HELP</a>
-    </li>
-    <li class="" id="dropdown-menu-https:--help.hydroshare.org-about-hydroshare-">
```

Comment:
so a link to about is going away?  Just making sure this is intentional

Reply:
Correct. Here is the discussion behind this change: https://github.com/hydroshare/hydroshare/issues/3242#issuecomment-472595918